### ISSUE TITLE
Ripple on Peatio

### ISSUE THREAD

**Issue by consynor** [peatio/peatio#618]
Has anyone successfully installed Ripple?

Id like to hear from you if you have

skype cubik929

**Comment by peatiotech** [comment]
Were you able to locate a developer for adding Ripple to your Peatio Exchange?

**Comment by CowboyKen** [comment]
Good Question.  I was curious about that too.

**Comment by scatterp** [comment]
yes i have it running

**Comment by CowboyKen** [comment]
Are you currently working with MeNagy (Mohammed)?

**Comment by scatterp** [comment]
I sent you an email

**Comment by amarian12** [comment]
i am also interesting of adding XRP  to my exchange, do we have a public implementation somewhere or i need to start from zero?

https://www.nlexch.com

**Comment by scatterp** [comment]
nothing public yet... maybe soon

**Comment by sushilraghav** [comment]
@consynor /// I can help you. I am having a vast experience in Peatio, Crypto Currency exchange and trading platforms with FIAT currency based on Bitcoin, litecoin, Ethereum, XRP etc, Core Wallets and Daemons. I am having expertise in ICO ie Initial Coin Offering full system development, White Paper creation with full architecture and high level documentation, smart contract writing and many more skills.

My Skype : sushilraghav and email is admin@rensoftglobal.com

**Comment by amarian12** [comment]
Yes, we did it for Ripple and Casinocoin new wallet( that’s ripple base) you can see it working here:

https://www.nlexch.com/

**Comment by sushilraghav** [comment]
@amarian12  good work on nlexch.com can you setup this for me ?

**Comment by amarian12** [comment]
I can help you with instructions

**Comment by sushilraghav** [comment]
My skype: sushilraghav

How much for the setup ?

Best Regards  |  Sushil Kumar Singh Raghav
Rensoft Technosys Pvt Ltd
B-2, Swasthya Vihar (Near Scope Minar)
Delhi - 110092

Phone: 011-43015807-907

**Comment by amarian12** [comment]
mail me at info@nlexch.com with what exactly you want to achieve

**Comment by sushilraghav** [comment]
I just need you to setup like nlexh.com



Best Regards  |  Sushil Kumar Singh Raghav
Rensoft Technosys Pvt Ltd
B-2, Swasthya Vihar (Near Scope Minar)
Delhi - 110092

Phone: 011-43015807-907

**Comment by sushilraghav** [comment]
And how much do you charge for adding MXN $ as currency to my existing
scatterp repo ?



Best Regards  |  Sushil Kumar Singh Raghav
Rensoft Technosys Pvt Ltd
B-2, Swasthya Vihar (Near Scope Minar)
Delhi - 110092

Phone: 011-43015807-907

**Comment by Apollo-IT** [comment]
I've already developed and deployed 3 exchanges based on Peatio.
I can add more coins and customize exchanges with peatio.
Also have experiences in wlox.
Feel free to message me any time.
**skype : live:apollo447722
mail : [apollo447722@outlook.com](mailto:apollo447722@outlook.com)**
Regards.

**Comment by CowboyKen** [comment]
Have you been able to make xrp or ETH work?

Ken Hayes Jr.
+1-415-287-0420
+54911 3501-6854

**Comment by Apollo-IT** [comment]
Yes, I've done ETH and ETC, XRP, XEM also
also can add any currency do you want

**Comment by CowboyKen** [comment]
And how much do you charge for such services? Or are you interested in building team on an exchange project and see where it goes?

Ken Hayes Jr.
+1-415-287-0420
+54911 3501-6854

**Comment by Apollo-IT** [comment]
Hi
business can be discussed on Mail or Chat
Which messenger do you use?

**Comment by Apollo-IT** [comment]
send your contact with mail or chat

**Comment by CowboyKen** [comment]
Hi


Ken Hayes Jr.
Telephone +1-415-287-0420
WhatsApp +54911 3501-6854
Skype: kenhayesjr420
The exchange is https://ArDax.com.ar it's a peatio install. I do most of the work, but we could make more progress together!

**Comment by Apollo-IT** [comment]
i sent contact request in skype

**Comment by CowboyKen** [comment]
I want to also declare that Nagy and Scatterp have both contributed and I could not have done it without them.

Ken Hayes Jr.
+1-415-287-0420
+54911 3501-6854

**Comment by pankaj9310** [comment]
I have installed ripple on peatio exchange. But address is not generating. Can anyone share config file of ripple.
Thanks in advance

**Comment by rockandruby** [comment]
@pankaj9310 ripple api changed thus u need to change ripple implementation in peatio. Now, if u want to generate address u need to bootstrap ripple server locally and then perform admin actions(like wallet_propose for address generation)

**Comment by pankaj9310** [comment]
@rockandruby Plz share your contact details. if you help me to integrate ripple on peatio then I will pay you.
Thanks,

**Comment by rockandruby** [comment]
@pankaj9310 thks) but I'm not interested in freelance. Just gave u a small advice.

**Comment by pankaj9310** [comment]
@rockandruby thank you, can you share me ripple rpc coin code. That will be very helpful for me. Thanks again

**Comment by rockandruby** [comment]
@pankaj9310 What do u mean by rpc coin code?
